src/containers/PermissionsApplication.jsx

```diff
@@ -34,7 +35,8 @@ export const completePermission = (
   return completedPermission
 }
 
-const PermissionsApplication = ({ match, t }) => {
+const PermissionsApplication = ({ match }) => {
+  const { t } = useI18n()
```

Comment:
Since you use `withAllLocales`, there is no need for `useI18n`, you can retrieve the `t` from the props just like before